python/ray/tests/autoscaler_test_utils.py

```diff
@@ -43,7 +43,7 @@ def matches(self, tags):
 class MockProcessRunner:
     def __init__(self, fail_cmds=None, cmd_to_callback=None, print_out=False):
         self.calls = []
-        self.cmd_to_callback = cmd_to_callback or {}  # type: Dict[str, Callable]
+        self.cmd_to_callback: Dict[str, Callable] = cmd_to_callback or {}
```

Comment:
Disclaimer: I am just some random person showing up here.

As the minimum version is now 3.9 and [PEP 585](https://www.python.org/dev/peps/pep-0585) is available. `typing.Dict, typing.Callable` and similar are deprecated.

Now we can (and should rather) use `from collections.abc import Callable` and the builtins dict instead.

```suggestion
        self.cmd_to_callback: dict[str, Callable] = cmd_to_callback or {}
```